dev/test/watch.ts

```diff
@@ -141,8 +141,7 @@ function snapshot(
   ref: DocumentReference,
   data: DocumentData
 ): QueryDocumentSnapshot {
-  const snapshot = new DocumentSnapshotBuilder();
-  snapshot.ref = ref;
+  const snapshot = new DocumentSnapshotBuilder(ref);
   snapshot.fieldsProto = ref.firestore._serializer!.encodeFields(data);
   snapshot.readTime = new Timestamp(0, 0);
   snapshot.createTime = new Timestamp(0, 0);
```

Comment:
Please also update test/typescript.ts to make sure the new types are what we expect. We may drop this file at some point, but we should keep it updated as long as it exists.

Reply:
Done.